app/controllers/support_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class SupportController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @form = SupportForm.new(new_params)
+  end
+
+  def create
+    @form = SupportForm.new(create_params)
+
+    if @form.valid? && @form.save
+      flash[:success] = { title: "Success", content: "Your support request has been submitted, our support team will get back to you shortly." }
```

Comment:
Could we take them to a dedicated 'success' page, [like this one](https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/early-career-teachers/NDY745/support-contacted), instead of redirecting back to the contact form with a flash message?

If that's too complicated, then could we redirect them to the ECT page they started from and show the flash message there, instead of on the contact form page?

Reply:
I can do this, though getting in the customisation related to what the ticket was for would add some complication. I'd suggest we do that in a separate ticket if necessary.